doc/governance/governance.rst

```diff
@@ -122,3 +122,62 @@ The following repositories are managed by the ros-controls PMC:
      - Marq Rasmussen
    * - https://github.com/ros-controls/.github
      - Not Yet Available
+
+Releases, Versioning, and Public API
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+As ros-controls PMC is independent of the ROS PMC we thrive to follow `its strategy <https://docs.ros.org/en/rolling/The-ROS2-Project/Contributing/Developer-Guide.html#quality-practices>`__ but have to adapt
+it according to our needs. This includes an asynchronous release cycle, where ros-controls claims to
+have a stable release for a ROS distro on **1st of October after an official ROS distribution release**.
+But ros2_control packages may be available via the ROS build farm earlier than this date.
+The stable release, called **released ros2_control version** from now, will be announced on `ROS discourse <https://discourse.openrobotics.org/c/ros-controls/ros-controls-announce-news/107>`__ on time.
+
+Versioning
+~~~~~~~~~~
+
+We will use the ROS-specific rules on top of ``semver's`` for versioning, but also adhere to some ros-controls-specific rules:
+
+* Major version increments (i.e. breaking changes) should not be made within a **released ros2_control version**.
+
+* ros2_control heavily relies on the usage of `pluginlib <https://index.ros.org/p/pluginlib/>`__. Therefore, we distinguish two types of compiled code: non-plugin code together with plugin base classes, and plugins itself.
+
+  * ABI of plugins may change at every release, i.e., also within **released ros2_control version**. Typically, they are built by the buildfarm and not intended to be linked against your code.
+  * For non-plugin code, the ABI is considered part of the public interface. Any change that requires recompiling dependent code is considered major (breaking) and must not be made within a **released ros2_control version**.
+
+Public API declaration
```

Comment:
I haven't changed this section from the ROS version. Open for discussion.

- Should we even install header files of plugins if we don't expect them to be API stable?
- Or install them, but clearly distinguish between public and private API?

Reply:
good question! I think pluginlib doesn't require headers, only the XML files and the base classes (if custom)